### ISSUE TITLE
Fix layout

### ISSUE THREAD

**Issue by Brunty** [Brunty/checker-services#3]
Tables are all uneven and annoying. Find a way to fix this.

**Comment by StyxOfDynamite** [comment]
Doesn't look as though Github flavoured markdown supports table formatting?

https://help.github.com/articles/organizing-information-with-tables/

**Comment by Brunty** [comment]
Yeah, I've been pondering how to solve this as it doesn't let you do much :/
